### DISCUSSION TITLE
Is there any UI (web) to test cron expressions that will work with Quartz

### DISCUSSION THREAD

**Discussion by eriveraa** [quartznet/quartznet#1676] [General]
I have seem many cron expressions that don´t work with quartz. However in those websites which generate them actually works.
So it would be nice if we can test the cron expression before scheduling jobs with quartz.
Any ideas?

**Comment by lahma** [comment]
[CronMaker](http://www.cronmaker.com/) uses Quartz and its format, it should be best place to start.
